test/e2e/flask/solana/common-solana.ts

```diff
@@ -1631,6 +1628,9 @@ export async function withSolanaAccountSnap(
             await mockGetAccountInfo(mockServer),
           ]);
         }
+        if (mockZeroBalance) {
+          mockList.push(await mockSolanaBalanceQuote(mockServer, true));
+        }
```

Comment:
we place this here to over-ride the default solana mocks, if this flag is passed in the spec